### ISSUE TITLE
[jshint] add which rule is being violated in messages

### ISSUE THREAD

**Issue by dhodgin** [madskristensen/WebEssentials2013#72]
Would it be possible to add which jshint rule is being violated in messages to help figure out which option is being violated. This would help the user to figure out if they want to turn it off if its not important or lookup the reason why the specific rule is being violated.

As a fairly new user to jshint I find it hard to sometimes know what rules certain messages are violating in the setup options.

Basically turn this:
![jshint1](https://f.cloud.github.com/assets/145013/1157933/664fed12-1faa-11e3-998b-d9cb191fa320.png)

Into this:
![jshint2](https://f.cloud.github.com/assets/145013/1157935/6e662e76-1faa-11e3-983e-e0a75f92cedd.png)

If jshint just feeds these messages back then I suppose it might not be possible but I think it would help users understand and therefor utilize jshint more often.
